### PR TITLE
Allow failures on --pre builds for travis and appveyor

### PR DESCRIPTION
**Pull request by hmaarrfk**
## Description
Apparently not yet possible for azure.
https://github.com/microsoft/azure-pipelines-tasks/issues/9302

At least we don't have *soooo* much red that it is super distracting.

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ environment:
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - PIP_FLAGS: --pre
```
Comment (jni) [inline .appveyor.yml]: Does this do anything? Or *will* it do anything?

Comment (jni) [reply .appveyor.yml]: Never mind, I just realised it's appveyor, not azure. I forgot we still had this! =P

**Comment by hmaarrfk** [comment]
why did travis just not run here? or at least, not show up?

**Comment by sciunto** [comment]
@hmaarrfk Can you try with a new PR to see...

**Pull request closed without merging**